webapp/lib/view.dart

```diff
@@ -500,161 +475,186 @@ class BatchRepliesConfigurationView extends PackageConfiguratorView {
         ..append(addsTagsContainer)
     );
   }
+}
+
+class TagListView extends BaseView {
+  List<TagView> tagElements;
+  DivElement _tagsContainer;
+  SpanElement _tagsActionContainer;
+  Function onTagChangedCallback;
 
-  SpanElement _addTagAction(List<String> tags) {
-    return new SpanElement()
+  TagListView(this.tagElements, Map<String, model.TagStyle> tagList, this.onTagChangedCallback, [bool tagsEditable = false]) {
```

Comment:
As above: suggest we rename this to `allTags` or `availableTags`

Reply:
Changed